README.md

```diff
@@ -1,4 +1,102 @@
-Takeaway Challenge
+# Takeaway Challenge
```

Comment:
- Great to have such an in-depth run through of your process.

- Would be useful to include your irb process - just to make it extra easy to run.